edi_835_parser/segments/claim.py

```diff
@@ -11,16 +11,23 @@ class Claim:
 	status = ClaimStatus()
 	charge_amount = Dollars()
 	paid_amount = Dollars()
+	patient_responsibility_amount = Dollars()
 
 	def __init__(self, segment: str):
 		self.segment = segment
 		segment = split_segment(segment)
 
 		self.identifier = segment[0]
-		self.marker = segment[1]
+		self.patient_control_number = segment[1]
 		self.status = segment[2]
 		self.charge_amount = segment[3]
 		self.paid_amount = segment[4]
+		self.patient_responsibility_amount = segment[5]
+		self.facility_type_code = segment[8]
```

Comment:
segments > 4 should use the get_element(segment,8) function. Specifically needed for Aetna.
```suggestion
		self.facility_type_code = get_element(segment, 8)
```